app/bundles/CoreBundle/Resources/views/Default/navbar.html.twig

```diff
@@ -53,8 +53,24 @@
             </a>
             <!--/ brand -->
          </li>
+{% include '@MauticCore/Helper/button.html.twig' with {
+        buttons: [
+            {
+            label: 'mautic.core.search_everything',
+			tooltip_placement: 'right',
+            variant: 'ghost',
+            icon: 'ri-search-line ri-xl',
+            icon_only: true,
+            attributes: {
+            'type': 'button',
+            'data-toggle': 'modal',
+            'data-target': '#gsearchModal',
+            'class': 'search-button'
+            }
+            }
+        ]
+        } %}
```

Comment:
```suggestion
        {% include '@MauticCore/Helper/button.html.twig' with {
            buttons: [
                {
                    label: 'mautic.core.search_everything',
                    tooltip_placement: 'right',
                    variant: 'ghost',
                    icon: 'ri-search-line ri-xl',
                    icon_only: true,
                    attributes: {
                    'type': 'button',
                    'data-toggle': 'modal',
                    'data-target': '#gsearchModal',
                    'class': 'search-button'
                }
                }
            ]
        } %}
```

Minor Indentation things.

Reply:
Do you use a specific formatter for Twig in VS Code?

These weird indentations happen every time when using the normal HTML formatter :/

Reply:
I am using PHPStrom for Mautic. I'm not sure about the VS Code formatter.